tests/conftest.py

```diff
@@ -271,7 +271,7 @@ def additional_test_data() -> dict[str, Any]:
         project_name="TEST-PROJECT-NAME4",
         primary_intervention_theme="TEST-PIT4",
         locations="TEST-LOCATIONS4",
-        postcodes="",  # no postcode == no region
+        postcodes=[],  # no postcode == no region
```

Comment:
interestingly, if you change this to None, the data comes out the same from the serialisation (postcodes field for this row has "" value), but the records come out in a different order (so the test fails). I'm not suggesting we change anything, it's just weird, and makes me uneasy about asserting things in tests based on record order, when in reality that record order is arbitrary